docs/data-review-request.md

```diff
@@ -0,0 +1,105 @@
+
+# Request for data collection review form
+
+**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**
+
+1) What questions will you answer with this data?
+
+TODO
+
+2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?
+
+Connection speed is important for Firefox feature prioritization / decision making and it can provide information essential for advancing a business objective such as supporting OKRs.
+
```

Comment:
In addition, a great browser experience depends very much on a reliable and speedy internet connection. There are many product features we can innovate, improve and customize based on our users internet speed. If we have a reliable, representative data set of internet speeds, we can inform our browser product decisions that improve the users experience on the internet. Moreover, we can use the data to inform public policy, ultimately improving the browsing experience for all users of the Internet.